darwin/.env.example

```
@@ -0,0 +1,30 @@
+# Apple App Store Signing Configuration
+# Copy this file to .env and fill in your actual values
+
+# App Store Connect - App's numeric ID (NOT your personal Apple ID email)
+APPLE_ID_NUMERIC="1234567890"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)